### PR TITLE
fix ideal printing

### PR DESCRIPTION
**Pull request by tthsqe12**

### CODE REVIEW

File: src/printing.jl [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    function Base.show(io::IO, mi::MIME"text/latex", x::$T)
+       AbstractAlgebra.show_via_expressify(io, mi, x)
+    end
+
+    function Base.show(io::IO, mi::MIME"text/html", x::$T)
+       AbstractAlgebra.show_via_expressify(io, mi, x)
+    end
+  end
+end
```
Comment (thofma) [inline src/printing.jl]: Should this be in AbstractAlgebra?

Comment (thofma) [reply src/printing.jl]: Maybe hide it behind `isdefined(AbstractAlgebra, Symbol("@enable_all..."))`, then we can move it to AA whenever we want.

Comment (tthsqe12) [reply src/printing.jl]: If you could explain why, when moved to AA, this code needs $(esc(T)), I would be grateful.

Comment (thofma) [reply src/printing.jl]: You should attend the juliacon workshop on metaprogramming.

**Pull request merged**